code/modules/antagonists/nukeop/equipment/borgchameleon.dm

```diff
@@ -90,6 +93,60 @@
 		user.filters = null
 		animation_playing = FALSE
 
+/obj/item/borg_chameleon/proc/fulp_borg_chameleon_menu(mob/living/silicon/robot/user) //Surrealaser's code
```

Comment:
Though it makes sense for you, as you are porting the proc and know where it comes from, for future users the name won't be very informative. Change it to something more descriptive of what this does.
Also, document it.